src/librustdoc/html/templates/STYLE.md

```diff
@@ -32,7 +32,7 @@ Askama templates support quite sophisticated control flow. To keep our templates
 simple and understandable, we use only a subset: `if` and `for`. In particular
 we avoid [assignments in the template logic][assignments] and [Askama
 macros][macros]. This also may make things easier if we switch to a different
-Jinja-style template system, like Askama, in the future.
+Jinja-style template system in future.
```

Comment:
```suggestion
Jinja-style template system in the future.
```

Reply:
I find the word redundant

Reply:
Apparently [this is a british vs american english difference](https://www.oxfordlearnersdictionaries.com/us/definition/english/future_1?q=future), where BE uses "in future", and AE uses "in the future".

Reply:
@est31 note that "in future" is used as "from now on" in the idioms section, which has a different meaning than the sentence here, which refers to "at a later time" instead.